src/e_lgamma.c

```diff
@@ -21,16 +21,19 @@
  * Method: call __ieee754_lgamma_r
  */
 
-#include <openlibm_math.h>
+#include <float.h>
 
+#include <openlibm_math.h>
 #include "math_private.h"
 
-DLLEXPORT double
+extern int signgam;
+
+double
 __ieee754_lgamma(double x)
 {
-#ifdef OPENLIBM_ONLY_THREAD_SAFE
```

Comment:
Yeah, figuring out a way to automate the update-and-rebase process would be a really good idea. Maybe something with subtree or sparse checkout could preserve the git history on just the libm parts of the upstream source?